### PR TITLE
CT: Add STATICCALL op

### LINKED ISSUE #383: CT: add specification of the STATICCALL instruction
The specification of the STATICCALL instruction in Tosca CT is missing.

### PR DESCRIPTION
**Pull request by facuMH**
This PR adds STATICCALL op to the specification.

This PR fixes #383

### CODE REVIEW

File: go/ct/spc/specification.go [hunk 1]
```diff
@@ -1900,26 +1900,26 @@ func rulesFor(i instruction) []Rule {
 	return res
 }
 
-func getCallRules() []Rule {
+func getCallAndStaticRules() []Rule {
```
Comment (HerbertJordan) [inline go/ct/spc/specification.go:1903]: nit: rename to `getCallAndStaticCallRules`

**Review by simonlechner — DISMISSED** [review]
Looks good.

File: go/ct/spc/specification.go [hunk 2]
```diff
@@ -1345,7 +1345,7 @@ func getAllRules() []Rule {
 
 	// --- CALL ---
```
Comment (HerbertJordan) [inline go/ct/spc/specification.go:1346]: nit: Update comment to "CALL and STATICCALL"

**Review by HerbertJordan — APPROVED** [review]
Looks good, thanks!

**Review by simonlechner — APPROVED** [review]

**Pull request merged**